extensions/amp-facebook-page/0.1/amp-facebook-page.js

```diff
@@ -0,0 +1,92 @@
+/**
+ * Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+import {getIframe, preloadBootstrap} from '../../../src/3p-frame';
```

Comment:
Looks like there's a linter error in Travis: https://travis-ci.org/ampproject/amphtml/jobs/337825128 You'll need to sort your import lines.